### PR TITLE
spicetify-cli: Add version 2.9.2

### PR DESCRIPTION
**Pull request by sitiom**
Relates to #7962

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
The manifest looks fine to me, but I think this should be in the [Main bucket](https://github.com/ScoopInstaller/Main).

File: bucket/spicetify-cli.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2.9.2",
+    "description": "Tool for customizing the Spotify client.",
+    "homepage": "https://spicetify.app/",
+    "license": "LGPL-2.1-only",
+    "notes": "Make sure to set 'spotify_path' to the directory of your Spotify installation in the Spicetify config",
+    "suggest": {
+        "Spotify": "spotify"
```
Comment (issaclin32) [inline bucket/spicetify-cli.json:8]:
```suggestion
        "Spotify": "extras/spotify"
```

**Comment by sitiom** [comment]
> The manifest looks fine to me, but I think this should be in the [Main bucket](https://github.com/ScoopInstaller/Main).

Makes sense.

> You can just link the issue by putting `closes #7962` in the comment, this will automatically close issue when the PR is merged.

I've put `relates` instead because the issue wasn't specifically requesting Spicetify but Spotify, which would be https://github.com/ScoopInstaller/Extras/pull/8119. But sure, I'll make it `closes`.

**Comment by sitiom** [comment]
Succeded by https://github.com/ScoopInstaller/Main/pull/3365.

File: bucket/spicetify-cli.json (hunk 1 above)
Comment (sitiom) [reply bucket/spicetify-cli.json:8]: Added in https://github.com/ScoopInstaller/Main/pull/3365.

**Pull request closed without merging**